setup.py

```diff
@@ -96,4 +96,5 @@ def _get_readme():
                         'pep8>=1.7.0',
                         'requests_mock>=1.0.0,<2.0',
                         'lxml>=3.6.0,<4.0',
+                        'keystoneauth1>=2.16,<3.0',
```

Comment:
I will remove this once we figure out the refactor as suggested in your later comments (removing `auth.py`)